DataFormats/L1Trigger/interface/BXVector.impl

```diff
@@ -1,18 +1,20 @@
 #include <vector>
 #include <iostream>
+#include <cassert>
 using namespace std;
```

Comment:
This line in a (file included in a) header file is problematic!
@mulhearn @rekovic @davidlange6